src/main/resources/db/changesets/20200625_initial_schema.yaml

```diff
@@ -0,0 +1,143 @@
+              name: name
+              type: text
+              constraints:
+                nullable: true
+                unique: false
+    - createTable:
+        tableName: tracked_resource
+        columns:
+          - column:
+              name: tracked_id
+              type: uuid
+              constraints:
+                primaryKey: true
+                nullable: false
+          - column:
+              name: resource_type_id
+              type: uuid
+              constraints:
+                references: resource_type(type_id)
+                foreignKeyName: fk_type_id
+                nullable: true
+          - column:
+              name: resource_uid
+              type: json
+              constraints:
+                nullable: true
+                unique: false
+          - column:
+              name: creation
+              type: timestamp
+              constraints:
+                nullable: true
+                unique: false
+          - column:
+              name: expireation
+              type: timestamp
+              constraints:
+                nullable: true
+                unique: false
+          - column:
+              name: state
+              type: text
+              constraints:
+                nullable: true
+                unique: false
+          - column:
+              name: wait_until
+              type: timestamp
+              constraints:
+                nullable: true
+                unique: false
+    - createTable:
+        tableName: label
+        columns:
+          - column:
+              name: tracked_id
+              type: uuid
+              constraints:
+                references: tracked_resource(tracked_id)
+                foreignKeyName: fk_tracked_id
+                primaryKey: true
+                nullable: false
+          - column:
+              name: key
+              type: text
+              constraints:
+                nullable: true
+                unique: false
+          - column:
+              name: value
+              type: text
+              constraints:
+                nullable: true
+                unique: false
+    - createTable:
+        tableName: cleanup_log
+        columns:
+          - column:
+              name: tracked_id
+              type: uuid
+              constraints:
+                references: tracked_resource(tracked_id)
+                foreignKeyName: fk_tracked_id
+                primaryKey: true
+                nullable: false
+          - column:
+              name: logtime
+              type: timestamp
+              constraints:
+                nullable: true
+                unique: false
+          - column:
+              name: status
+              type: text
+              constraints:
+                nullable: true
+                unique: false
+          - column:
+              name: description
+              type: text
+              constraints:
+                nullable: true
```

Comment:
Maybe?  My preference would be that all log entries require a description, but I wouldn't be surprised if we find some obscure scenarios where it is not needed?

Reply:
I think this is optional and should be nullable?

Reply:
I think we'd end up writing dummy descriptions that might as well be null. So, I agree that nullable is fine.
Like, what's the description for when the resource is already cleaned up? That's all captured in the status.